### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Marius1311**
**IMPORTANT: Please search among the [Pull request](../pulls) before creating one.**

## Title
Update the docs to reflect push for more general markov-state modelling for single-cell data

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature (non-breaking change which adds functionality)

## Description
I change the description of the CellRank structure to emphasize that CellRank is more than just RNA velo + trans. similarity - it's a general workflow for Markov-chain based analysis of single-cell data. 

## Closes

closes #555

### CODE REVIEW

**Comment by Marius1311** [comment]
Hi @michalk8, let me know what you think of this, I can then go ahead and also modify the readme.

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-see [Manno18]_ and [Bergen20]_. In short, CellRank models cellular dynamics as a
-Markov chain, where transition probabilities are computed based on **RNA velocity and transcriptomic similarity**,
-taking into account **uncertainty in the velocities** and the stochastic nature of cell fate decisions.
-The Markov chain is coarse-grained into a set of macrostates which represent initial and terminal states,
-as well as transient intermediate states using Generalized Perron Cluster Cluster Analysis (G-PCCA) [GPCCA18]_,
-implemented in the novel `pyGPCCA`_ package. For each transient cell, i.e. for each cell that's not assigned to a
-terminal state, we then compute its fate probability of it reaching any of the terminal states.
-We show an example of such a fate map in the figure above, which has been computed using the data
-of [Panc19]_.
+**CellRank** is a toolkit to uncover cellular dynamics based on Markov state modelling of single-cell data,
```
Comment (michalk8) [inline docs/source/index.rst:10]: modelling -> modeling (to be consisteny with EN_US)

**Review by michalk8 — CHANGES_REQUESTED** [review]
Looks good, just minor things. Once this is merged, please don't forget to update the README in the notebooks repo.

**Comment by Marius1311** [comment]
Thanks for your comments Mike! Will go over them today.

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,22 +1,24 @@
 |PyPI| |Bioconda| |Downloads| |CI| |Notebooks| |Docs| |Codecov|
 
-CellRank - Probabilistic Fate Mapping using RNA Velocity
+CellRank for directed single-cell fate mapping
 ========================================================
```
[the remark this answers is not in the document]
Comment (Marius1311) [reply docs/source/index.rst]: done.

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
-as well as transient intermediate states using Generalized Perron Cluster Cluster Analysis (G-PCCA) [GPCCA18]_,
-implemented in the novel `pyGPCCA`_ package. For each transient cell, i.e. for each cell that's not assigned to a
-terminal state, we then compute its fate probability of it reaching any of the terminal states.
-We show an example of such a fate map in the figure above, which has been computed using the data
-of [Panc19]_.
+**CellRank** is a toolkit to uncover cellular dynamics based on Markov state modelling of single-cell data,
+taking into account the stochastic nature of cellular fate decisions. It contains two main modules:
+**kernels** compute cell-cell transition probabilities and **estimators** generate hypothesis based on these
+transition probabilities. Our kernels work with a variety of input data including RNA velocity (see [Manno18]_
+and [Bergen20]_), transcriptomic similarity, spatial proximity and pseudotime. Our ``VelocityKernel`` takes into
```
[the remark this answers is not in the document]
Comment (Marius1311) [reply docs/source/index.rst]: done.

File: docs/source/index.rst (hunk 1 above)
Comment (Marius1311) [reply docs/source/index.rst]: done.

**Comment by Marius1311** [comment]
@michalk8, I added quite a few changes, please take another look.

**Review by michalk8 — APPROVED** [review]

**Review by michalk8 — COMMENTED** [review]
1 question: do you also indent to update the `README.rst`? I think they should be the same.

**Comment by Marius1311** [comment]
> 1 question: do you also indent to update the `README.rst`? I think they should be the same.

Yes! Will do that as well.

**Comment by Marius1311** [comment]
Are tests still failing because of matplotlib?

**Comment by michalk8** [comment]
> Are tests still failing because of matplotlib?

Yes, they are fixed in #556 , so I think after the update you can merge.
Btw, since you're just modifying docs/readme, you should use `[ci skip] <you commit msg>`.

**Comment by Marius1311** [comment]
> > Are tests still failing because of matplotlib?
> 
> Yes, they are fixed in #556 , so I think after the update you can merge.
> Btw, since you're just modifying docs/readme, you should use `[ci skip] <you commit msg>`.

Uh, good point, thanks!

**Comment by Marius1311** [comment]
This can be merged, also includes the `README.rst` now.

**Comment by Marius1311** [comment]
Now just need to update the README in the notebook repo.

**Comment by Marius1311** [comment]
`git commit [ci skip] -m "COMMIT MESSAGE"` gives `zsh: bad pattern: [ci`

**Comment by michalk8** [comment]
> git commit [ci skip] -m "COMMIT MESSAGE" gives zsh: bad pattern: [ci

It should be in the commit message: `git commit -m "[ci skip] foo bar baz"`. Feel free to merge if it's ready.

**Comment by Marius1311** [comment]
> > git commit [ci skip] -m "COMMIT MESSAGE" gives zsh: bad pattern: [ci
> 
> It should be in the commit message: `git commit -m "[ci skip] foo bar baz"`. Feel free to merge if it's ready.

Got it, thanks.

**Pull request merged**